src/avatar/Avatar.js

```diff
@@ -105,21 +110,28 @@ const Avatar = ({
     <Component
       onPress={onPress}
       onLongPress={onLongPress}
-      style={[
+      style={StyleSheet.flatten([
         styles.container,
         { height, width },
         rounded && { borderRadius: width / 2, overflow: 'hidden' },
```

Comment:
Hi, I was planning to Open PR for this issue. I saw that you're working for other stuff. Maybe you can change it quickly!!

The `borderRadius` should not be add on Parent component. This is causing UI bug that edit icon isn't showing properly


A workaround for now

![image](https://user-images.githubusercontent.com/18099464/48663398-a0a5ca00-eab5-11e8-8857-96b62ff4c537.png)

Reply:
Yes i saw it! :D About to change that now

Reply:
And I was straggling to find the issue. Anyway awesome library